src/tex/ms.tex

```diff
@@ -249,7 +249,7 @@ \section{Conclusions}
 Our work shows that carefully curated benchmarks can provide a more nuanced understanding of the capabilities of \glspl{llm} in the chemical sciences.
 Importantly, our findings also illustrate that more focus is required in developing better human-model interaction frameworks, given that models cannot estimate their limitations.
 
-While our findings indicate many areas for further improvement of \gls{llm}-based systems, it is also important to realize that clearly defined metrics have been the key to the progress of many fields of \gls{ml}, such as computer vision.
+While our findings indicate many areas for further improvement of \gls{llm}-based systems, such as for the ReAct systems (more discussion in \cref{sec:react-environment}), it is also important to realize that clearly defined metrics have been the key to the progress of many fields of \gls{ml}, such as computer vision.
```

Comment:
```suggestion
While our findings indicate many areas for further improvement of \gls{llm}-based systems, such as for agents (more discussion in \cref{sec:react-environment}), it is also important to realize that clearly defined metrics have been the key to the progress of many fields of \gls{ml}, such as computer vision.
```

Reply:
I'd tend to keep the language here simple - in the end, it is a chemistry journal for the average chemist :)